client/src/sockets/enums.js

```diff
@@ -0,0 +1,30 @@
+import {
+  ItemActionTypes,
+  CommentActionTypes
+} from 'modules/list/components/Items/model/actionTypes';
+
+export const ItemsEvents = Object.freeze({
+  ADD_SUCCESS: ItemActionTypes.ADD_SUCCESS,
+  ARCHIVE_SUCCESS: ItemActionTypes.ARCHIVE_SUCCESS,
+  CLEAR_VOTE_SUCCESS: ItemActionTypes.CLEAR_VOTE_SUCCESS,
+  CLONE_SUCCESS: ItemActionTypes.CLONE_SUCCESS,
+  DELETE_SUCCESS: ItemActionTypes.DELETE_SUCCESS,
+  RESTORE_SUCCESS: ItemActionTypes.RESTORE_SUCCESS,
+  SET_VOTE_SUCCESS: ItemActionTypes.SET_VOTE_SUCCESS,
+  UPDATE_SUCCESS: ItemActionTypes.UPDATE_SUCCESS
+});
+
+export const ItemStatusType = Object.freeze({
```

Comment:
Do we need separate enum for that? Could we add them to ItemsActionTypes. They are used in the same way as actionTypes?

Reply:
ItemActionTypes describes actions related to the items, like deleting archiving, etc.

BUSY and FREE are not an action, they represent STATE of the item so I don't think that mixing them all together is a good idea.